### PR TITLE
Offline fix offline panel swicth and update ngeo

### PR DESCRIPTION
**Pull request by ger-benjamin**
Finish GSLUX-89

### CODE REVIEW

File: geoportailv3/static/js/maincontroller.js [hunk 1]
```diff
@@ -473,8 +473,10 @@ app.MainController = function(
         return this.ngeoNetworkStatus.offline;
       }, () => {
       if (this.ngeoNetworkStatus.offline) {
-        this.closeSidebar();
-        this.layersOpen = true;
+        if (!this['layersOpen'] && !this['mymapsOpen']) {
+          this.closeSidebar();
+          this['layersOpen'] = true;
```
Comment (fgravin) [inline geoportailv3/static/js/maincontroller.js]: Are you sure we want to open the panel while offline ?

**Comment by ger-benjamin** [comment]
Thanks @fgravin 
Now I doesn't open it if it is closed.

I also fix a always true if below in the code.

File: geoportailv3/static/js/maincontroller.js [hunk 2]
```diff
@@ -473,8 +473,10 @@ app.MainController = function(
         return this.ngeoNetworkStatus.offline;
       }, () => {
       if (this.ngeoNetworkStatus.offline) {
-        this.closeSidebar();
-        this.layersOpen = true;
+        if (this.sidebarOpen() && !this['layersOpen'] && !this['mymapsOpen']) {
+          this.closeSidebar();
+          this['layersOpen'] = true;
```
Comment (fgravin) [inline geoportailv3/static/js/maincontroller.js]: Do we really favour `this['layersOpen']` over `this.layersOpen` ?

Comment (ger-benjamin) [reply geoportailv3/static/js/maincontroller.js]: That always looks like this below in the code, so I though there is a reason for. I didn't try with the dot notation

Comment (fgravin) [reply geoportailv3/static/js/maincontroller.js]: Ok i guess it's the way to do the `@export` thing.
Please merge.

Comment (ger-benjamin) [reply geoportailv3/static/js/maincontroller.js]: That's not nice, but that's coherent... :-/

**Pull request merged**